client/extensions/woocommerce/index.js

```diff
@@ -265,9 +266,19 @@ export default async function ( _, addReducer ) {
 	// Add pages that use the store navigation
 	getStorePages().forEach( function ( storePage ) {
 		if ( config.isEnabled( storePage.configKey ) ) {
-			addStorePage( storePage, ( context, next ) =>
-				createStoreNavigation( context, next, storePage )
-			);
+			// Store deprecation would redirect most store pages to dashboard
+			if ( isStoreRemoved && ! ( '/store/:site' === storePage.path ) ) {
+				page( storePage.path, addTracksContext, ( context ) => {
+					// Tracks MC stats by path /store/products/:site -> store-products-:site
+					const trackName = storePage.path.substr( 1 ).replaceAll( '/', '-' );
+					context.store.dispatch( bumpStat( 'calypso_store_post_sunset', trackName ) );
```

Comment:
Will result in sending a GET request to MC stats with  `calypso_store_post_sunset` stat group and the route path transformed to a more readable form (slashes are removed in MC stat) as the stat name.

E.x: `/store/products/:site` will produce `http://pixel.wp.com/g.gif?v=wpcom-no-pv&x_calypso_store_post_sunset=store-products-%3Asite`